server/Gemfile

```diff
@@ -27,11 +27,6 @@ gem 'net-scp'
 gem 'net-ssh'
 gem 'net-sftp'
 
-# javascript
-# gem 'libv8', platforms: :ruby
-gem 'execjs'
-gem 'therubyracer', platforms: :ruby
```

Comment:
This doesn't work in dockerland, although it appears to in OSX and Windows. Putting this in a 'docker' group and updating.